Change the default prompt to the one which login traditionally uses.

diff -uNr Linux-PAM-0.75/libpam/pam_private.h Linux-PAM-0.75/libpam/pam_private.h
--- Linux-PAM-0.75/libpam/pam_private.h	Mon Feb  5 01:50:41 2001
+++ Linux-PAM-0.75/libpam/pam_private.h	Tue Jul 10 00:14:23 2001
@@ -264,7 +265,17 @@
 
 /* Definition for the default username prompt used by pam_get_user() */
 
-#define PAM_DEFAULT_PROMPT "Please enter username: "
+#define PAM_DEFAULT_PROMPT "login: "
+
+/*
+ * pam_system_log default ident/facility..
+ */
+
+#define PAM_LOG_STATE_DEFAULT {      \
+    PAM_LOG_STATE_IDENT,     \
+    PAM_LOG_STATE_OPTION,    \
+    PAM_LOG_STATE_FACILITY   \
+}
 
 /*
  * include some helpful macros
